protocol/communities_messenger_token_permissions_test.go

```diff
@@ -578,6 +578,11 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestBecomeMemberPermissions(
 	err = <-waitOnCommunityEncryptionErrCh
 	s.Require().NoError(err)
 
+	// Make sure Bob gets the new community with the new permission
+	community, err = s.owner.GetCommunityByID(community.ID())
+	s.Require().NoError(err)
+	s.advertiseCommunityTo(community, s.bob)
```

Comment:
No need to advertise community explicitly, `CreateCommunityTokenPermission` should do that already.
```suggestion
	_, err := WaitOnMessengerResponse(
		s.bob,
		func(r *MessengerResponse) bool {
			return len(r.Communities()) > 0
		},
		"no community",
	)
	s.Require().NoError(err)
```

Reply:
I think I tried that, but it didn't work, but that was when I was writing the test, so I assume I had still issues in the code itself.

Now it works fine. Updated. Thanks.